### PR TITLE
kim2021re

### PR DESCRIPTION
**Pull request by sunniesuhyoung**
I wasn't able to locally compile with 'make' so I formatted the camera-ready version and checked compiling in overleaf ([view link](https://www.overleaf.com/read/hymxpkszkzgq)). Hope everything works smoothly!

### CODE REVIEW

File: kim2021re/journal/packages.tex [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+\usepackage{amssymb}
+\usepackage{bm}
+\usepackage{float}
+\usepackage{bbm}
+\usepackage{hhline}
+\usepackage{todonotes}
+\usepackage{color, colortbl}
+\usepackage{multirow}
+\usepackage{subcaption}
+\usepackage{enumitem}
```
Comment (koustuvsinha) [inline kim2021re/journal/packages.tex:43]: Remove the `enumitem` import

File: kim2021re/journal/packages.tex [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+%\usepackage{natbib} % Remove for ReScience
+
+%\hypersetup{colorlinks=true, citecolor=MidnightBlue, urlcolor=blue} % Remove for ReScience
+\definecolor{light-gray}{gray}{0.9}
+
+
+% Added by me
+\usepackage{tablefootnote}
+\usepackage{enumerate}
```
Comment (koustuvsinha) [inline kim2021re/journal/packages.tex:52]: Also remove the `enumerate` import

**Review by koustuvsinha — COMMENTED** [review]
Ok, this took a while to debug, but I narrowed down the issue to the use of `enumerate` - loading both `enumitem` and `enumerate` package breaks the compilation. Loading neither compiles (possibly rescience.cls also loads enumerate on its own). Please remove both imports and check, should work and formatting should be okay too.

**Comment by sunniesuhyoung** [comment]
I removed both packages and compilation worked on my end!

**Comment by koustuvsinha** [comment]
Thanks for the change! Compilation works now.

@sashavor can you review the paper once? LGTM on my end

**Comment by sashavor** [comment]
PDF is looking good!

**Pull request merged**